shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedProcessorServiceImpl.java

```diff
@@ -1,19 +1,59 @@
 package com.redhat.service.smartevents.shard.operator.v2;
 
+import java.util.List;
+
 import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
 
 import com.redhat.service.smartevents.infra.v2.api.models.dto.ProcessorDTO;
+import com.redhat.service.smartevents.shard.operator.v2.converters.ManagedProcessorConverter;
+import com.redhat.service.smartevents.shard.operator.v2.providers.NamespaceProvider;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedProcessor;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
 
 @ApplicationScoped
 public class ManagedProcessorServiceImpl implements ManagedProcessorService {
 
+    @Inject
+    KubernetesClient kubernetesClient;
+
+    @Inject
+    NamespaceProvider namespaceProvider;
+
     @Override
     public void createManagedProcessor(ProcessorDTO processorDTO) {
-        // @TODO
+        String expectedNamespace = namespaceProvider.getNamespaceName(processorDTO.getBridgeId());
```

Comment:
This looks to relate to https://issues.redhat.com/browse/MGDOBR-1251 and not https://issues.redhat.com/browse/MGDOBR-1252 (which is the subject of this PR). Have you chosen to merge the work into a single PR? If so can you comment the JIRA or something so it is clear.

Reply:
Implementation of [MGDOBR-1251](https://issues.redhat.com/browse/MGDOBR-1251) was done my mocking service layer. I had added the implementation of these methods in [MGDOBR-1252](https://issues.redhat.com/browse/MGDOBR-1252) to incorporate Rob's review comment.